firmware/shared/controls/tvFactor.h

```diff
@@ -1,35 +1,39 @@
 #pragma once
-#include <vector>
 #include <algorithm>
 #include <tuple>
 #include "../util/mappers/lookup_table.h"
 
 namespace ctrl{
 
-template <typename T>
-T EnableTorqueVectoring(T steering_angle, T torque_vectoring_switch) {
-    if (torque_vectoring_switch > 0)
-        return steering_angle;
-    else return 0;
+bool EnableTorqueVectoring(bool torque_vectoring_switch) {
+    if (torque_vectoring_switch == true) {
+        return true;
+    } else {
+        return false;
+    }
 }
 
 template <typename T>
-T CreateTorqueVectoringFactor(T steering_angle) {
-    T absolute_steering_angle = std::abs(steering_angle);
+T CreateTorqueVectoringFactor(T steering_angle, bool torque_vectoring_switch) {
```

Comment:
a better name for the bool is `enable_torque_vectoring` since "switch" does not indicate what true/false means